packages/harmony/src/storybook/Welcome.stories.mdx

```diff
@@ -0,0 +1,166 @@
+      component={
+        <Flex alignItems='center' gap='xl'>
+          <Text variant='display' size='s'>
+            Ag
+          </Text>
+          <Flex
+            h={32}
+            w={32}
+            alignItems='center'
+            justifyContent='center'
+            css={{ background: 'rgba(90, 73, 202, 0.10)' }}
+          >
+            <Text>32</Text>
+          </Flex>
+          <Flex direction='column'>
+            <Box
+              h={24}
+              w={24}
+              css={(theme) => ({ background: theme.color.neutral.neutral })}
+            />
+            <Box
+              h={24}
+              w={24}
+              css={(theme) => ({ background: theme.color.neutral.n400 })}
+            />
+            <Box
+              h={24}
+              w={24}
+              css={(theme) => ({ background: theme.color.neutral.n150 })}
+            />
+          </Flex>
+          <Flex direction='column' gap='m'>
+            <Paper h={24} w={24} shadow='near' borderRadius='xs' />
+            <Paper h={24} w={24} shadow='near' borderRadius='s' />
+            <Paper h={24} w={24} shadow='near' borderRadius='2xl' />
+          </Flex>
+          <IconNote color='default' />
+        </Flex>
+
+      }
+    />
+    <InformationBox
+      title='Components'
+      description='Detailed information and guidelines on each component.'
+      href='/?path=/docs/components-welcome--documentation'
+      css={{ overflow: 'hidden' }}
+      component={
+        <Flex
+          gap='l'
+          alignItems='center'
+          wrap='wrap'
+        >
+          <Button size='small' variant='primary'>
+            Button
+          </Button>
+          <SelectablePill label='Pill' size='large' />
+          <TextInput
+            label='Display name'
+            placeholder='express yourself ✨'
+            _isFocused
+          />
+        </Flex>
+      }
+    />
+
+  </Flex>
+</Unstyled>
+
+## Our Principles
+
+<Unstyled>
+  <Flex gap='3xl' mv='3xl'>
+    <InformationBox
+      title='Unity'
+      description='Uniting designers & developers with a unified language'
+      component={<IconUnity />}
+    />
+    <InformationBox
+      title='Consistency'
+      description='Uniting designers & developers with a unified language'
+      component={<IconConsistency />}
+    />
+    <InformationBox
+      title='Scalable'
+      description='Uniting designers & developers with a unified language'
+      component={<IconScalable />}
+    />
+    <InformationBox
+      title='Reusable'
+      description='Aim to increases production and speed'
+      component={<IconReusable />}
+    />
+  </Flex>
+</Unstyled>
+
+## For Designers & Developers
+
+Empowering creativity with streamlined resources. Dive into our collaborative tools.
+
+<Unstyled>
+  <Flex gap='3xl' mv='3xl'>
+    <InformationBox
+      title='Figma'
+      description='Our public Figma library on Community.'
+      component={<IconFigma css={{ width: 50 }} />}
```

Comment:
does this need a link, or is it just a placeholder for now?

Reply:
Yeah it should use a link good call, will just link to our internal Figma for now